R/cat_add_data.R

```diff
@@ -7,15 +7,113 @@
 #' sources.
 #'
 #' @param x an object of class [elic_cat].
-#' @param mechanism
+#' @param data_source either a [`data.frame`][base::data.frame] or
+#' [`tibble`][tibble::tibble], a string with the path to a _csv_ or _xlsx_ file,
+#' or anything accepted by the [read_sheet()][googlesheets4::read_sheet]
+#' function.
+#' @param mechanism integer indicating what mechanism the data belongs to.
 #' @inheritParams elic_cont_add_data
 #'
+#' @section Data Format:
+#'
+#' For each mechanism, data are expected to have five columns, built as follows:
+#' * The first column of the data should hold the names of the experts. The name
+#' of each expert should be repeated as many times as there are impact levels
+#' and sites considered in the elicitation
+#' (i.e. each expert should appear \eqn{number\ of\ levels
+#' \cdot number\ of\ sites} times).
+#' * The second column should be the names of the levels considered in the
+#' eliciation. Each level should be repeated as many times as there are sites
+#' considered.
+#' * The third column should hold the names of the sites considered in the
+#' study. The name of each site should be repeated as many times as the number
+#' of levels.
+#' * The fourth column should be the experts confidence levels (given in
+#' percents). Experts should each give one confidence level per site, so each
+#' confidence level should be repeated for all levels at each site.
+#' * The final column should be the estimates of each expert.
```

Comment:
Line 34: Maybe add "the estimates of each expert" for each site and confidence level?